dlp/snippets/risk.py

```diff
@@ -383,6 +383,167 @@ def callback(message: google.cloud.pubsub_v1.subscriber.message.Message) -> None
 # [END dlp_k_anonymity]
 
 
+# [START dlp_k_anonymity_with_entity_id]
+import time  # noqa: I100,  F811, E402
+from typing import List  # noqa: E402, F811
+
+import google.cloud.dlp  # noqa: I100, F811, E402
```

Comment:
issue: is this a correct import? You use `google.cloud.dlp_v2` in line 423

Reply:
that's how its used in other samples, yet I have updated the same